src/reconstruction/ppmc_cuda.h

```diff
@@ -1,21 +1,44 @@
 /*! \file ppmc_cuda.h
  *  \brief Declarations of the cuda ppm kernels, characteristic reconstruction
  * version. */
-#ifdef CUDA
-  #ifdef PPMC
 
-    #ifndef PPMC_CUDA_H
-      #define PPMC_CUDA_H
+#ifndef PPMC_CUDA_H
+#define PPMC_CUDA_H
 
-      #include "../global/global.h"
+#include "../global/global.h"
 
-/*! \fn void PPMC(Real *dev_conserved, Real *dev_bounds_L, Real *dev_bounds_R,
- int nx, int ny, int nz, int n_ghost, Real dx, Real dt, Real gamma, int dir)
- *  \brief When passed a stencil of conserved variables, returns the left and
- right boundary values for the interface calculated using ppm. */
-__global__ void PPMC_cuda(Real *dev_conserved, Real *dev_bounds_L, Real *dev_bounds_R, int nx, int ny, int nz,
-                          int n_ghost, Real dx, Real dt, Real gamma, int dir, int n_fields);
+/*!
+ * \brief Computes the left and right interface states using PPM with limiting in the characteristic variables and
+ * characteristic tracing. Used for the CTU and SIMPLE integrators
+ *
+ * \param[in] dev_conserved The conserved variable array
+ * \param[out] dev_bounds_L The array of left interfaces
+ * \param[out] dev_bounds_R The array of right interfaces
+ * \param[in] nx The number of cells in the X-direction
+ * \param[in] ny The number of cells in the Y-direction
+ * \param[in] nz The number of cells in the Z-direction
+ * \param[in] dx The length of the cells in the `dir` direction
+ * \param[in] dt The time step
+ * \param[in] gamma The adiabatic index
+ * \param[in] dir The direction to reconstruct. 0=X, 1=Y, 2=Z
+ */
+__global__ void PPMC_CTU(Real *dev_conserved, Real *dev_bounds_L, Real *dev_bounds_R, int nx, int ny, int nz, Real dx,
+                         Real dt, Real gamma, int dir);
 
-    #endif  // PPMC_CUDA_H
-  #endif    // PPMC
-#endif      // CUDA
+/*!
+ * \brief Computes the left and right interface states using PPM with limiting in the characteristic variables. Used for
+ * the VL (Van Leer) integrators
```

Comment:
It would be useful if the description of these two functions also included what is different about the PPM.

Reply:
I added more info in [3b192d3](https://github.com/cholla-hydro/cholla/pull/300/commits/3b192d3f6329764beab332983fcea790e20d91a0). Could you take a look and let me know if that's what you meant?

Reply:
I think I've addressed it now. It's still a little vague but it's hard to get into details without just reciting an entire paper or the code. It has all the citations and an overall description now anyway.

Reply:
Much better, thank you!